appdata.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/data/com.github.raibtoffoletto.litteris.appdata.xml.in b/data/com.github.raibtoffoletto.litteris.appdata.xml.in
+index 15dcb67..5376f65 100644
+--- a/data/com.github.raibtoffoletto.litteris.appdata.xml.in
++++ b/data/com.github.raibtoffoletto.litteris.appdata.xml.in
+@@ -28,15 +28,4 @@
+   <url type="bugtracker">https://github.com/raibtoffoletto/litteris/issues</url>
+   <url type="help">https://raibtoffoletto.github.io/litteris</url>
+ 
+-  <releases>
```

Comment:
I don't think this patch is needed as the release tag is required. You also need the OARS tag https://odrs.gnome.org/oars

Reply:
Yesterday I was getting errors saying that the release was duplicated.... so I created the patch. Today (after upgrading everything to the tag 0.1.6) that's gone...
So I altered the patch to include the OARS (will be included by default the next release I have, no point of making a release just for a line).